DigitalLearningSolutions.Data/Services/NotificationService.cs

```diff
@@ -87,5 +93,84 @@ public async Task SendUnlockRequest(int progressId)
+                    <a href='{finaliseUrl}'>here</a>. Only one evaluation can be submitted per completion.</p>";
+            var textActivityCompletionInfo = completionStatus == 2
+                ? "To evaluate the activity and access your certificate of completion, click here."
+                : "If you haven't already done so, please evaluate the activity to help us to improve provision " +
+                  "for future delegates by clicking here. Only one evaluation can be submitted per completion.";
+
+            if (numLearningLogItemsAffected == 1)
+            {
+                htmlActivityCompletionInfo +=
+                    "<p>This activity is related to <b>1</b> planned development log action in another activity " +
+                    "in your Learning Portal. This has automatically been marked as complete.</p>";
+                textActivityCompletionInfo +=
+                    " This activity is related to 1 planned development log action in another activity " +
+                    "in your Learning Portal. This has automatically been marked as complete.";
+            }
+            else if (numLearningLogItemsAffected > 1)
+            {
+                htmlActivityCompletionInfo +=
+                    $@"<p>This activity is related to <b>{numLearningLogItemsAffected}</b> planned development log actions " +
+                    "in other activities in your Learning Portal. These have automatically been marked as complete.</p>";
+                textActivityCompletionInfo +=
+                    $@" This activity is related to {numLearningLogItemsAffected} planned development log actions " +
+                    "in other activities in your Learning Portal. These have automatically been marked as complete.";
+            }
+
+            if (progressCompletionData.AdminEmail != null)
+            {
+                htmlActivityCompletionInfo +=
+                    "<p><b>Note:</b> This message has been copied to the administrator(s) managing this activity, for their information.</p>";
+                textActivityCompletionInfo +=
+                    " Note: This message has been copied to the administrator(s) managing this activity, for their information.";
+            }
+
+            const string emailSubjectLine = "Digital Learning Solutions Activity Complete";
+            var builder = new BodyBuilder
+            {
+                TextBody = $@"Dear {progress.DelegateFirstName ?? "Digital learning Solutions delegate"},
+                    You have completed the Digital Learning Solutions learning activity - {progressCompletionData.GetCourseNameWithoutAverageTimeInfo()}.
+                    {textActivityCompletionInfo}",
+                HtmlBody = $@"<body style=""font-family: Calibri; font-size: small;"">
+                                <p>Dear {progress.DelegateFirstName ?? "Digital learning Solutions delegate"},</p>

```

Comment:
Incorrect capitalisation for Learning in "Digital learning Solutions delegate" for both version

Reply:
fixed